bot/modules/owner.py

```diff
@@ -113,15 +114,17 @@ def __init__(self, bot):
             await ctx.send('**`SUCCESS`**')
             logger.info('<{0}> UnLoaded successfully.'.format(cog))
 
-    @commands.command(name='reload', hidden=True)
+    @commands.command(name='reload')
     @commands.is_owner()
     async def cog_reload(self, ctx, *, cog: str):
         """Command which Reloads a Module.
         Remember to use dot path. e.g: cogs.owner"""
 
+        cogfix = 'modules.'+cog
```

Comment:
We should be able to come up with a better name than cogfix. how about full_cog_path?
```suggestion
        full_cog_path = 'modules.'+cog
```